bincapz.go

```diff
@@ -70,6 +70,7 @@ func main() {
 	errFirstHitFlag := flag.Bool("err-first-hit", false, "exit with error if scan source has matching capabilities")
 	ociFlag := flag.Bool("oci", false, "Scan an OCI image")
 	omitEmptyFlag := flag.Bool("omit-empty", false, "Omit files that contain no matches")
+	frequencyUpgradeFlag := flag.Bool("frequency-upgrade", true, "increase file risk score based on frequency heuristics")
```

Comment:
Maybe `upgrade-severity` is a better flag name?

Reply:
How about --quantity-increases-risk ?

Reply:
I think that works. Since this defaults to `true` it's not something that needs to be written out all that often.